platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -88,6 +88,92 @@ export class StudyMetadata extends Metadata {
     return this._displaySets.slice();
   }
 
+  _createDisplaySetsForSeries(
```

Comment:
the bulk of the work originally carried on by the `createDisplaySets` method but to a single series object